applications/nrf5340_audio/README.rst

```diff
@@ -1224,13 +1224,13 @@ Complete the following steps to test the unidirectional CIS mode for one gateway
    The playback volume increases for both headsets.
 #. Press the **VOL-** button on the gateway.
    The playback volume decreases for both headsets.
-#. Press the **PLAY/PAUSE** button on one of the headsets.
-   The playback stops for the given headset and continues on the other one.
+#. Press the **PLAY/PAUSE** button on any one of the devices.
+   The playback stops for both headsets and the streaming state for all devices is set to paused.
 #. Press the **RESET** button on the gateway.
    The gateway resets and the playback on the unpaused headset stops.
    After some time, the gateway establishes the connection with both headsets and resumes the playback on the unpaused headset.
-#. Press the **PLAY/PAUSE** button on one of the paused headsets.
-   The playback resumes in sync with the other headset.
+#. Press the **PLAY/PAUSE** button on any one of the devices.
+   The playback resumes in both headsets.
```

Comment:
in -> on

Reply:
Disagree, the audio is in the headset, not on the headset